makeflow/src/makeflow.c

```diff
@@ -1102,6 +1119,27 @@ static void show_help_run(const char *cmd)
 	printf(" %-30s Format for monitor logs.                    (default %s)\n", "   --monitor-log-fmt=<fmt>", DEFAULT_MONITOR_LOG_FORMAT);
 }
 
+static void makeflow_remove_sandbox(const char *makeflow_log_file){
```

Comment:
Nice!  Since this function deals entirely with the log file, call it `makeflow_log_clean` and move it to `makeflow_log.c`